fabric-loot-api-v3/src/main/java/net/fabricmc/fabric/api/loot/v3/LootTableEvents.java

```diff
@@ -95,6 +100,16 @@ private LootTableEvents() {
 			listener.onLootTablesLoaded(resourceManager, lootManager);
 		}
 	});
+	/**
```

Comment:
```suggestion

	/**
```
Forgot to include this in the previous review